prep-javascript-variables/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullname = 'Kevin Bowler';
+    console.log('Fullname:', fullname);
+
+    var numberofpets = '5';
+    console.log('Numberofpets:', numberofpets);
+
+    const likesVegetables = false;
+    Boolean(likesVegetables);
+    console.log('likesVegetables:', likesVegetables);
+
+    var nothing = 'null';
+    console.log('nothing:', nothing);
```

Comment:
For numbers, booleans, and `null`, you'll want to leave out the quotes so that the JavaScript engine doesn't interpret them as strings.

Reply:
![Prep-javascript-variables-2](https://user-images.githubusercontent.com/43438792/150015931-daf4bdee-b3fd-4a52-bf44-af05a403fd74.png)